x/gravity/keeper/keeper_delegate_key.go

```diff
@@ -13,18 +15,22 @@ import (
 ///////////////////////////
 
 // GetOrchestratorValidator returns the validator key associated with an orchestrator key
-func (k Keeper) GetOrchestratorValidator(ctx sdk.Context, orch sdk.AccAddress) (validator stakingtypes.Validator, found bool, err error) {
+func (k Keeper) GetOrchestratorValidator(ctx context.Context, orch sdk.AccAddress) (validator stakingtypes.Validator, found bool, err error) {
 	valAddr := sdk.ValAddress(orch)
 	if valAddr == nil {
 		return validator, false, fmt.Errorf("nil valAddr")
 	}
 
 	if err := sdk.VerifyAddressFormat(orch); err != nil {
-		ctx.Logger().Error("invalid orch address")
+		liblog.FromSDKLogger(k.Logger(ctx)).Error("invalid orch address")
```

Comment:
You're not including the actual error in the logs. Use `WithError(err)` here and all other cases like this.